pages/reference/OS/secure-boot-and-full-disk-encryption/generic-x86-64-gpt.md

```diff
@@ -0,0 +1,137 @@
+
+
+#### Using {{ $names.cloud.lower }}
+
+1. Add a device to your fleet. You should be presented with the following new option:
+
+<img alt="Add a device modal with secure boot and full disk encryption modal" src="/img/common/sb-fde/x86/add-device-modal-sb-fde-x86.webp" width="100%">
+
+2. As mentioned, there are [important considerations and caveats][considerations-caveats] when enabling secure boot and full disk encryption. You will be presented with the following disclaimer when enabling the feature:
+
+<img alt="Warning for considerations when enabling secure boot and full disk encryption" src="/img/common/sb-fde/x86/sb-fde-x86-disclaimer.webp" width="100%">
+
+ You can optionally surpress the disclaimer by selecting "Don't show me this warning again for this device type".
+
+ You can then proceed to download the configured image.
+
+ #### Using {{ $names.cli.lower }} 
+
+{{ $names.cli.upper }} versions `16.2.0` or newer allow you to [configure](https://docs.balena.io/reference/balena-cli/#os-configure-image) a balenaOS installer image to opt-in secure boot by using the following command:
+
+````
+balena os configure <image> --secureBoot --version <version> --fleet <fleetName> --device-type generic-amd64
+
+````
+
+### Flash the image to a USB key
+
+The configured image should be flashed to a USB key.
+
+- Insert the USB key to the host machine.
+- Write the balenaOS file you downloaded to the USB key. We recommend using Etcher.
+- Wait for writing of balenaOS to complete.
+- Remove the USB key from the host machine.
+- Insert the freshly flashed USB key into the Generic x86_64 (GPT).
+
+### Provision the device
+
+Once the image is ready, the device needs to be configured in secure boot setup mode. This depends on the UEFI/BIOS implementation, but in general there are the following steps to consider:
+
+- Reset to the default UEFI configuration.
+- Make sure the device is configured to boot in UEFI mode, for example by checking the Compatibility Support Module (CSM) used for Master Boot Record (MBR) booting is disabled.
+- Change the boot options to allow booting only from the USB installer/flasher device and the main storage, and choose the USB as first boot option. The flasher will then set the UEFI configuration to boot from the main storage before rebooting.
+- Disable restoring factory keys. Some systems default to restoring factory (Microsoft) keys rather than using the balena keys that will be installed during setup.
+- Enable secure boot.
+- Reset device to setup mode
+
+On booting in setup mode, the installer will enroll the keys into UEFI variables and encrypt the disks using the TPM device. Note that enrolling the keys manually via the UEFI setup application, while possible on some systems, is not currently supported by balenaOS as the installer’s bootloader is not signed.
```

Comment:
I would end the sentence at `... is not currently supported by balenaOS.` without elaborating. We know it's broken even beyond the missing signature on the default bootloader, but we're not exactly sure why. Anyway, it is not supported.